src/gc/gc.cpp

```diff
@@ -31551,7 +31551,7 @@ void gc_heap::descr_generations_to_profiler (gen_walk_fn fn, void *context)
                 assert (seg == hp->ephemeral_heap_segment);
                 assert (curr_gen_number0 <= max_generation);
                 //
-                if ((curr_gen_number0 == max_generation))
+                if (curr_gen_number0 == max_generation)
```

Comment:
No, Not unless we want to stop passing -Wparentheses-equality flag to clang compiler.
With the flag this is reported as an error